README.md

```diff
@@ -65,6 +65,7 @@ route = routific.get_route()
 
 puts route.status # => "success"
 puts route.total_travel_time # => 29
```

Comment:
Let's set the `polyline` option in the example as well.

Reply:
@rogertsui0923 pls review - after this I need help figuring out how to build the gem.